apps/etterlatte-vedtaksvurdering-kafka/src/main/kotlin/regulering/services/VedtakService.kt

```diff
@@ -25,4 +26,9 @@ class VedtakServiceImpl(private val vedtakKlient: HttpClient, private val url: S
         runBlocking {
             vedtakKlient.post("$url/api/vedtak/upsert/$behandlingId").body()
         }
+
+    override fun fattVedtak(behandlingId: UUID): Vedtak =
+        runBlocking {
+            vedtakKlient.post("$url/api/vedtak/fattvedtak/$behandlingId").body()
```

Comment:
dette endepunktet forventer ett bearer token med claimet `navIdent` som den setter som saksbehandler. Blir egentligen ikke riktig for vårt case.